libraries/ESP8266WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -217,14 +257,16 @@ public StaticRequestHandler<ServerType> {
         if (SRH::_cache_header.length() != 0)
             server.sendHeader("Cache-Control", SRH::_cache_header);
 
-        server.sendHeader("ETag", etag);
+        if ((server._eTagEnabled) && (_eTagCode.length() > 0)) {
+            server.sendHeader("ETag", _eTagCode);
+        }
 
         server.streamFile(f, mime::getContentType(SRH::_path), requestMethod);
         return true;
     }
 
 protected:
-    uint8_t _ETag_md5[16];       
+    String _eTagCode; // eTag as used in http header for this single file
```

Comment:
Can you please clarify this comment ?

Reply:
ETag code calculated for this file as used in http header - including quotes.
changed the comment but what to say more ?